cartridge/stateboard.lua

```diff
@@ -155,6 +155,12 @@ local function set_vclockkeeper_impl(replicaset_uuid, instance_uuid, ordinal, vc
 
     local vclockkeeper = box.space.vclockkeeper:get({replicaset_uuid})
     if ordinal ~= (vclockkeeper and vclockkeeper.ordinal) then
+        if vclockkeeper.instance_uuid == instance_uuid
+        and vclock == nil
+        then
+            -- No update needed
+            return true
+        end
```

Comment:
i think more beautiful to keep this `if` branch separate with additional cond `ordinal ~= (vclockkeeper and vclockkeeper.ordinal)`